tests/ctst/features/iam-policies/IAMUser.feature

```diff
@@ -0,0 +1,82 @@
+Feature: IAM Policies for IAM Users
+  This feature allows you to create and attach IAM policies for IAM users.
+  IAM users should have the permissions to perform the actions that they are granted in their IAM policies.
+
+  @2.6.0
+  @PreMerge
+  @IAM-Policies-IAMUsers
+  Scenario Outline: User is not authorized to perform the actions without IAM policy
+    Given an existing bucket "<bucketName>" "without" versioning
+    And we "<haveObject>" an existing object "<objectName>" "without" version-Id ""
+    And a IAM_USER type
+    When the user tries to perform "<ifS3Standard>" "<action>" on the bucket "<haveObject>" the object
```

Comment:
I am not sure to understand this sentence, plus the "haveObject" should not be needed, it can be retrieved from previous steps with a default behavior

```suggestion
    When the user tries to perform a "<ifS3Standard>" "<action>" on the bucket

Reply:
I can confirm this has been [fixed](https://github.com/scality/Zenko/pull/1799/files#diff-284f11d6681baa8186a18ff40a18801ef2f353d3d23a6d8be5d52e3367ed92c9R5) in my PR